kubernetes-crew/terraform/variables.tf

```diff
@@ -45,39 +45,51 @@ variable "task_schedules" {
   description = "Schedule configuration for automated tasks"
   type = object({
     health_check = optional(object({
-      start_time = string
+      start_time = optional(string, "09:00")  # Set a default start time
```

Comment:
Shouldn't the start time needs to be in MM/DD/YYYY HH:mm
As scheduled tasks ?
This question is related to all the relevant vars which has start_time

@Kubiya-Barak